Library/Formula/hunspell.rb

```diff
@@ -18,6 +20,8 @@ def install
     system "make"
     ENV.deparallelize
     system "make", "install"
+
+    pkgshare.install %w[tests]
```

Comment:
Can just do `pkgshare.install "tests"`. No need for an array unless there's multiple things to install.